core/templates/dev/head/components/forms/schema_editors/SchemaBasedChoicesEditorDirective.js

```diff
@@ -37,7 +37,11 @@ oppia.directive('schemaBasedChoicesEditor', [
         'schema_based_choices_editor_directive.html'),
       restrict: 'E',
       compile: NestedDirectivesRecursionTimeoutPreventionService.compile,
-      controller: ['$scope', function($scope) {
+      controller: ['$scope', '$window', function($scope, $window) {
```

Comment:
@tjiang11 I wanted to disable the choices for language recognition in browsers other than chrome. So, I modified the ``isDisabled()`` when the schema was used for speech recognition and the browser did not support speech recognition. I tested this locally and it worked fine!

Reply:
Sorry, I'm a little confused what these choices are referring to. Is there any way you can take a screenshot of this? Is it possible for me to change the language I want to the speech recognition to be in from the learner view?

Reply:
There are choices for language in speech recognition. I wanted them to be enabled in chrome since only chrome supports ``webkitSpeechRecognition`` and other browsers don't.
Here are the screenshots for the same:
![screenshot from 2018-02-28 00-08-51](https://user-images.githubusercontent.com/22434689/36747779-ec344e9c-1c1b-11e8-9652-9a4c4c3cfb8c.png)
![screenshot from 2018-02-28 00-09-37](https://user-images.githubusercontent.com/22434689/36747789-f245d8fa-1c1b-11e8-82f7-4c9f97783091.png)
The first image is in a Chrome browser and the second one is in Firefox.

Reply:
Okay, so in the Firefox screenshot, it looks as if the language is set to English and unalterable. But this isn't actually the case, since speech recognition is completely disabled, in which case we just shouldn't even show this field. Also though, this doesn't really make sense from a creator perspective. I could be creating an exploration in Firefox but expect learners using Chrome to play my exploration, so we should not be disabling and enabling the feature based on what browser the creator is using.

Reply:
Yes, that is right @tjiang11. I will remove the disable feature!